README.md

```diff
@@ -1,2 +1,119 @@
-# neo4j-http
-A simple ruby HTTP client for Neo4j
+# Neo4j::Http
+
+The `Neo4j::Http` gem provides `Neo4j::Http::Client` as a thin wrapper around the [Neo4j HTTP API](https://neo4j.com/docs/http-api/current/).
```

Comment:
I wonder if it is worth explaining who needs this, and what the difference is between the HTTP API and the deprecated REST API? Maybe even highlight that this isn't a native driver that requires OS dependencies? We know the benefits but I think a README being the first thing users see is where we can "sell" it